scss/grid/_layout.scss

```diff
@@ -97,6 +97,11 @@
             text-overflow: none;
         }
 
+        .k-grouping-row .k-icon {
+            margin-left: addTwo(-1*$cell-padding-x, 7px);
```

Comment:
I think that there should be used $icon-size variable so icons could be centered in all cases.
If the header [k-group-cell width](https://www.screencast.com/t/TjCqs6jER) is ($icon-size*2), then "7px" can be replaced by ($icon-size/2).